### PR TITLE
You know the legion core nerf Citadel did? Yeah i fucking hate it.

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Reverts legion cores back to our modular solution. Because the Citadel one sucks.

## Why It's Good For The Game

Our modular solution was just fine and made legion cores way less viable on PVP, while also not punishing miners on lavaland. The Citadel solution has a fucking massive oversight, and punishes lavaland miners while giving traitor miners on station an edge. The problem here is, since the miner cannot be slowed down, he needs to literally reach stamcrit to be non-lethally disabled - a massive blow to security which normally uses disablers, which are only effective because they slow the target down.

Basically it's a shit solution that makes fighting megafauna harder for no reason, and removes a get out of jail card from extremely damaged miners, while also making legion cores still very damn useful on PVP.

## Changelog
:cl:
balance: Legion cores are now actually good again (while on lavaland, at least).
/:cl:

### CODE REVIEW

**Comment by Akrilla** [comment]
thank god
